tests/cargo-ui/stubbing-flag/src/main.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! This tests that the --enable-stubbing and --harness flags flow from Kani driver to Kani compiler
```

Comment:
Nit: I'd use `kani-driver` and `kani-compiler` (with backquotes in their names and the flags) so it's clear that we're talking about those crates. Other than that, great comments!

Reply:
Done!